docker/alpine.Dockerfile

```diff
@@ -127,3 +127,5 @@ COPY --from=ublock-build /etc/linkding/uBlock0.chromium uBlock0.chromium/
 RUN mkdir -p chromium-profile && chown -R www-data:www-data chromium-profile
 # enable snapshot support
 ENV LD_ENABLE_SNAPSHOTS=True
+
+LABEL org.opencontainers.image.source="https://github.com/sissbruecker/linkding"
```

Comment:
Thanks! Since the Dockerfiles contain multiple build stages that are used to generate an image (`linkding` and `linkding-plus`), I assume that the `LABEL` either needs to be set for both, or on the `linkding` build stage, which is the basis for `linkding-plus`.

Reply:
Oh you're right. I just assumed the last stage was the published one. I'll edit my commit.